### ISSUE TITLE
MpRester.query() is replaced with MpRester.summary.search()

### ISSUE THREAD

**Issue by karthiCk396** [materialsproject/pymatgen#2690]
**Describe the bug**
Before I used the MpRester.query() to download and access the all elements in the Material project but it changed to MpRester.summary.search() so I could pass the mongoDB query to download and access all the elements with properties = ['cif] 

Can someone suggest a command or query to download and access the all elements in the periodic table as cif file (using python /pymatgen).

Thanks in advance

**Comment by munrojm** [comment]
See https://matsci.org/t/how-do-you-access-the-cif-data-in-the-new-api/44850

**Comment by karthiCk396** [comment]
@munrojm Thank you for reply. This will help me definitely. Do I able to access all the elements of periodic table in single query instead of accessing the single element ?

**Comment by munrojm** [comment]
If you would like to query structures containing any element, simply remove the `elements` argument and only pass `fields` to `search`. It will default to querying structure data for every material in the database.

**Comment by karthiCk396** [comment]
Thank you @munrojm . It's working.

**Comment by karthiCk396** [comment]
@munrojm I have another doubt . How do I get the band gap value for the each element from the structured data of material

**Comment by karthiCk396** [comment]
I have used the below query to access the all elements with structure and band_gap fields.

results = mpr.summary.search(elements = ['Fe'],fields= ['structure',"band_gap"]) and converted into CIF file. After converting into the CIF file I have resued the cif file and trying to access the structure and bandgap. But I'm only able to access the structure but I'm not able to access the bandgap

and I used the function to get structure . struct1 = struct.get_structures()[0]

can anyone help me out in getting the bandgap from the structured data ?

**Comment by munrojm** [comment]
To clarify, you want the element projected band gap for each material?

**Comment by karthiCk396** [comment]
Yes @munrojm . I figured it out. Thank you for your time and help.
